sql/internal/sqlx/sqlx.go

```diff
@@ -20,6 +20,18 @@ func ValidString(s sql.NullString) bool {
 	return s.Valid && s.String != "" && strings.ToLower(s.String) != "null"
 }
 
+// ScanOne scans one record and closes the rows at the end.
+func ScanOne(rows *sql.Rows, dest ...interface{}) error {
```

Comment:
This should error out if there are more than 1 row, should't it?

Reply:
I tried to keep the same behavior as `sql.Row`.

Reply:
I was thinking about [database/sql](https://pkg.go.dev/database/sql#DB.QueryRow). It errors if there is more than one row in the result set.

Reply:
Sorry, it behaves exactly like you said.